google/cloud/forseti/services/dao.py

```diff
@@ -1639,7 +1639,7 @@ def delete(self, model_name):
         if model_name in self.sessionmakers:
             del self.sessionmakers[model_name]
         with self.modelmaker() as session:
-            session.query(Model).filter(Model.handle == model_name).delete()
+            session.query(Model).filter(Model.name == model_name).delete()
```

Comment:
It's hard to see this, but from the comment change above, how do we handle whether to delete by handle or to delete by name?

Reply:
We are no longer changing it in the dao.py, here is what we are doing:

1. get model by handle or name
2. delete that model by handle, this will be reverted